docs/src/main/sphinx/release/release-405.md

```diff
@@ -199,7 +233,8 @@
 * Add support for aggregation, join, and `ORDER BY ... LIMIT` pushdown. ({issue}`15365`)
 * Add support for `DELETE`. ({issue}`15365`)
 * Add schema, table, and column name length checks. ({issue}`15365`)
-* Add full type mapping for Redshift types. ({issue}`15365`)
+* Add full type mapping for Redshift types, which can be disabled with the
```

Comment:
Missing period at end of sentence.

Reply:
Also, maybe rephrase as "Add full type mapping for Redshift types. The previous behavior can be restored via the `redshift.use-legacy-type-mapping` configuration property"